openlibrary/templates/type/edition/view.html

```diff
@@ -1,9 +1,9 @@
 $def with (page)
 
 $ work = page.works and page.works[0] or None
-$ book_title = page.get('title') or work.title
-$ work_title = work.title or page.get('title')
-$ book_subtitle = page.subtitle if page.get('subtitle') else work.subtitle if work and work.get('subtitle') else None
+$ book_title = page.get('title', '') if page.title else work.title if work else ''
```

Comment:
SUGGESTION: Put parentheses around the clauses so that a reader won't have to wonder about operator precedence.